extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -54,6 +54,14 @@ export const ACTION_TYPE = {
   DISMISS: 'dismiss',
 };
 
+/**
+ * @enum {string}
+ */
+export const PROMPT_TRIGGER = {
+  ACTION: 'action',
+  FORCE_PROMPT_ON_NEXT: 'force-prompt-on-next',
```

Comment:
Thank you for making the change!! 'force-prompt-on-next' is no difference than the initial prompt. I don't think we need it. What about we use a boolean here instead?

Reply:
@zhouyx Can we discuss this offline tomorrow? I'm a little confused why we would use a boolean 🤔 . That being said, I wouldn't be opposed to it if it makes sense. 😄